### PR TITLE
Add an alert dialog before starting games

### PR DESCRIPTION
**Pull request by InfernoCoder**
I have done the GCI task of "Add an alert dialog before starting games. [MAL - Android]" here. ([Issue #105](https://github.com/systers/malaria-app-android/issues/105) ).

### CODE REVIEW

File: malaria-app-android/src/main/java/com/peacecorps/malaria/activities/NewHomeActivity.java [hunk 1]
```diff
@@ -92,7 +100,9 @@ public void onClick(View view) {
         return new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                startActivity(new Intent(NewHomeActivity.this, MythFactGame.class));
+                if (sharedPreferences.getBoolean("showHelpMythFactGame", true))
```
Comment (chhavip) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/activities/NewHomeActivity.java]: Put in strings.xml

File: malaria-app-android/src/main/java/com/peacecorps/malaria/activities/NewHomeActivity.java [hunk 2]
```diff
@@ -92,7 +100,9 @@ public void onClick(View view) {
...
+                    showHelpDialog(2);
```
Comment (chhavip) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/activities/NewHomeActivity.java]: Use integer.XML to define these integers

File: malaria-app-android/src/main/java/com/peacecorps/malaria/activities/NewHomeActivity.java [hunk 3]
```diff
@@ -101,8 +111,63 @@ public void onClick(View view) {
         return new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                startActivity(new Intent(NewHomeActivity.this, RapidFireGame.class));
+                if (sharedPreferences.getBoolean("showHelpRapidFireGame", true))
```
Comment (chhavip) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/activities/NewHomeActivity.java]: Same

File: malaria-app-android/src/main/res/layout/game_help_dialog.xml [hunk 4]
```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/dialogMedGroup"
+    android:layout_width="fill_parent"
+    android:layout_height="wrap_content"
+    android:background="@drawable/dialog_background">
+
```
Comment (chhavip) [inline malaria-app-android/src/main/res/layout/game_help_dialog.xml]: Remove extra lines throughout the XML file

**Comment by InfernoCoder** [comment]
@chhavip All changes done

**Comment by chhavip** [comment]
Thanks for making the changes!

**Pull request closed without merging**